README.md

```diff
@@ -41,7 +41,7 @@ Set to 0.0 to disable this effect.
 - `CenterScale` (default: 0.2) -- scale of the circle in the center, relative to note scale.
 - `ArrowScale` (default: 0.6) -- scale of the note arrow, relative to note scale.
 - `UIOpacity` (default: 1.0) -- Overall opacity for the UI.  Multiplicative with opacity/alpha of other colors.
-- `CanvasOffset` (default: `[0.0, 0.0, 16.0]`) -- control `[x, y, z]` position of bottom center point of the visualization canvas.  
+- `CanvasOffset` (default: `[0.0, 0.0, 16.0]`) -- control `[x, y, z]` position of bottom center point of the visualization canvas.

```

Comment:
Two trailing spaces here are intended

Reply:
I don't recall touching that :|
Will revert